tool/src/test/java/org/datacommons/tool/TestUtil.java

```diff
@@ -68,4 +74,28 @@ public static String readStringFromPath(Path filePath) throws IOException {
     File file = new File(filePath.toString());
     return FileUtils.readFileToString(file, StandardCharsets.UTF_8);
   }
+
+  public static void assertHtmlFilesAreSimilar(String expected, String actual) {
+    Parser htmlParser = Parser.htmlParser();
+    Document expectedHtml = htmlParser.parseInput(expected, "expected");
+    Document actualHtml = htmlParser.parseInput(actual, "actual");
+    LinkedList<Element> expectedElements = new LinkedList<>(expectedHtml.children());
+    LinkedList<Element> actualElements = new LinkedList<>(actualHtml.children());
+    while (!expectedElements.isEmpty() && !actualElements.isEmpty()) {
+      Element expectedElement = expectedElements.poll();
+      Element actualElement = actualElements.poll();
+      expectedElements.addAll(expectedElement.children());
+      actualElements.addAll(actualElement.children());
```

Comment:
I wonder about append all the actual and expected ownTexts into a single long string and comparing them?  Would the output of that be more readable than doing piecemeal (e.g., first element is an addition, there are extra elements, etc)?

Reply:
I like that idea, updated!

Reply:
Thanks. Maybe we can append *all* expectedElements and actualElements and then compare them?

Then we can get rid of the two assertTrue() statements below?

(Feel free to submit :) )